tests/unit/wordpress_mock.py

```diff
@@ -360,7 +362,8 @@ def list_files(self, path: str) -> typing.List[str]:
         return file_list
 
     def remove_path(self, path: str, recursive: bool = False) -> None:
-        """Mock method for :meth:`ops.charm.model.Container.remove_path`."""
+        # Reraise documentation for mocked path removal can be ignored.
```

Comment:
WOuldn't it be worth noting that in the docstring so users of the function know that this re-raise can occur?